CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+## Version 1.0.0-RC1
```

Comment:
I strongly suggest to tag `v1.0.0-rc.1`.
Some (most?) of semver tools would say that `1.0.0-RC1` > `1.0.0-RC1`.
This issue happened on OTel .NET 😉 

The `1.0.0-rc.1` syntax is mentioned (suggested?) in https://semver.org/